src/Apps/Auction/Components/RegistrationForm.tsx

```diff
@@ -232,7 +150,13 @@ const OnSubmitValidationError: React.FC<{
       !formikProps.isSubmitting &&
       !formikProps.isValid
     ) {
-      cb(Object.values(formikProps.errors))
+      const clonedErrors = Object.assign({}, formikProps.errors)
+      const addressErrors = clonedErrors.address
+      delete clonedErrors.address
+
+      const errors = Object.assign({}, clonedErrors, addressErrors)
```

Comment:
We're flattening the errors here so we end up with a flat array of error messages to send via our analytics event.

Reply:
If both are arrays you can use the same `Object.assign` spread syntax as with objects: `[...clonedErrors, ...addressErrors]` #minor

Reply:
> I guess this could be a spread instead?

Nope. TypeScript doesn't seem to like that /shrug